var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -84,6 +90,8 @@ class Hip(CMakePackage):
             when='@{0}'.format(d_version)
         )
     for d_version, d_shasum in [
+        ('5.0.2',  '34decd84652268dde865f38e66f8fb4750a08c2457fea52ad962bced82a03e5e'),
```

Comment:
Why are the rocclr and opencl resources downloaded both here and in the hip-rocclr package? Is there some reason this can't be a dependency on hip-rocclr?